src/components/features/language-selector/LanguageSelector.tsx

```diff
@@ -20,7 +20,7 @@ export const LanguageSelector = () => {
   const classes = useStyles();
   const router = useRouter();
 
-  const languageNames = new Intl.DisplayNames([locale || 'en'], {
+  const languageNames = new Intl.DisplayNames([], {
```

Comment:
Should we also apply those changes to the other templates?

Reply:
We don't have any errors for the other templates, BUT I will do a short review of the code to sanity check that.

Reply:
Thanks! I assume we would see the same errors if we used a similar `LanguageSelector` API, but since we did not get Sentry errors it might not be the case.

Reply:
@cf-remylenoir interestingly, it does occur on the other templates BUT it isn't a breaking issue because we render the language selector differently so it's never triggered as a hydration mismatch, see the console vs server output here:

![image](https://user-images.githubusercontent.com/8332893/210544486-cd24716e-6f96-4677-9def-a8a53cc79616.png)